micro-rdk/src/common/conn/server.rs

```diff
@@ -277,7 +277,7 @@ pub struct ViamServer<'a, C, T, CC, D, L> {
     app_connector: C,
     app_config: AppClientConfig,
     app_client: Option<AppClient<'a>>,
-    webrtc_manager: WebRTCConnectionManager,
+    incoming_connection_manager: IncomingConnectionManager,
```

Comment:
Potentially just `ConnectionManager`.

Reply:
+1 for `connection_manager` and `ConnectionManager`

Reply:
I think I changed my mind. It really is only handling inbound connections - ones we accepted.